x/oracle/client/cli/tx.go

```diff
@@ -47,9 +53,10 @@ func GetCmdPostPrice(cdc *codec.Codec) *cobra.Command {
 	}
 }
 
+// GetCmdAddOracle cli command for create new oracle.
 func GetCmdAddOracle(cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:     "add-oracle [nominee_key] [denom] [oracle_address]",
+		Use:     "add-oracle [nomineeKey] [assetCode] [oracle_address]",
 		Example: "dncli oracle add-oracle wallet1a7280dyzp487r7wghr99f6r3h2h2z4gk4d740m eth_usdt wallet1a7260dyzp487r7wghr99f6r3h2h2z4gk4d740k",
 		Short:   "Create a new oracle",
```

Comment:
Help msg isn't correct. That Tx doesn't "create an oracle", it rather adds it to assetCode